llvm/lib/Target/X86/X86PreTileConfig.cpp

```diff
@@ -237,11 +237,15 @@ void X86PreTileConfig::collectShapeInfo(MachineInstr &MI) {
 }
 
 bool X86PreTileConfig::runOnMachineFunction(MachineFunction &MF) {
+  X86MachineFunctionInfo *X86FI = MF.getInfo<X86MachineFunctionInfo>();
```

Comment:
Just found we have `set/hasHasVirtualTileReg` interfaces. We can remove it and replace with `X86FI->getAMXProgModel() == AMXProgModelEnum::ManagedRA` too.